packages/blade/docs/guides/Intro.stories.mdx

```diff
@@ -61,5 +61,4 @@ This package is under **maintenance** and it won't have any major releases. It w
 
 # ⌛️ Current State
 
-Here's a glimpse of where we are currently in our journey of building the design system.
-![phases-of-blade](https://user-images.githubusercontent.com/11384858/186387746-6233169f-3b1b-4d81-9f6e-7308dea7b273.png)
+For the current state of things please refer to the left hand sidebar. We're actively working on adding more components, utilities and docs. Please reach us on Slack at `#platform_fe_core` for any queries or create an issue on the GitHub repo.
```

Comment:
IDEA: For current state we can create a storybook recipes in a grid with all the components.

Reply:
Will increase the scope 😬

Reply:
Yeah we can keep it in backlog. Anyway we need a kitchen sink kind of story. not a huge effort :p

Reply:
:godmode: